osquery/carver/carver.cpp

```diff
@@ -147,6 +147,7 @@ Carver::~Carver() {
 }
 
 void Carver::start() {
+  setThreadName("carver");
```

Comment:
`InternalRunnable`s all have a `name_` value, would it be possible to poll what that value is and use it to name the thread instead of manually naming each?